README.md

```diff
@@ -55,12 +55,7 @@ time="2022-03-07T13:02:22Z" level=info msg="Found number +491234567890 and added
 time="2022-03-07T13:02:24Z" level=info msg="Started Signal Messenger REST API"
 ```
 
-6. Use the following snippet to get a group's `id`:
```

Comment:
Nice one, getting the group is not needed since version `0.13.0` of signalbot. So this was a leftover from versions before that one.